### ISSUE TITLE
Additions to the Chief Engineer's tools.

### ISSUE THREAD

**Issue by PetMudstone** [unitystation/unitystation#4127]
## Description
The CE's advanced tools (Experimental welding tool, Jaws of Life, Hand Drill) could use a few more features.

#### Jaws of Life and Hand Drill should have toggleable modes.
Right now the Jaws of Life has both crowbar and wirecutter traits and the hand drill has both screwdriver and wrench traits. They both have these dual traits active at the same time. While this works alright most of the time, it does cause odd behavior, such as when trying to use the hand drill on girders. 

You may intend to disassemble a girder after unsecuring it using the screwdriver trait but the wrench trait overrides it and makes you resecure the girder. 

Toggling modes on the jaws of life and hand drill should update the sprite, show a message, and perhaps even play a sound.

#### Jaws of Life could force open doors.
In /tg/, the Jaws of Life are able to force open doors, even powered ones, if they are used on them after a few seconds. If this behavior is added, I think it would be best if it was an individual component as other items (such as the Fire Axe and Crowbar) are able to force open doors (though not all can force open powered doors). 

#### Experimental Welding Tool needs to generate fuel.
The Experimental Welding Tool should generate it's own fuel over time as in /tg/.

- [x] Jaws of Life and Hand Drill toggleable modes.
- [x] Jaws of Life force open doors.
- [ ] Experimental Welding Tool regenerates fuel.

**Comment by NoooneyDude** [comment]
Only the last issue remains. Experimental Welding Tool should regenerate fuel.

**Comment by PerfectTangent** [comment]
as a general suggestion, what if the Experimental Welding Tool regenerates fuel by being charged at a charging port?
